src/actions/vm.js

```diff
@@ -288,6 +293,26 @@ export function setVmSessions ({ vmId, sessions }) {
   }
 }
 
+export function setVmSnapshots ({ vmId, snapshots }) {
```

Comment:
Why are these two action creators here and not in `VmSnapshots/actions.js`?

Reply:
Because `setVmSnapshots` are used not only within `VmSnapshot` component. `addVmSnapshot` I moved.